components/trial-bar/button.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import Button from '~/components/button';
+import { TRIAL_BAR_BUTTON } from '../../utilities/constants';
+
+// This component accepts the same props as Button.
```

Comment:
This is a doc site matter, so it's not very obvious, but this comment should follow the way that component descriptions should to be written to be picked up by `react-docgen`. I think that requires the comment it to in multi-line quote comment on the line directly before the class/function declaration. A markdown URL to the Button page would be good https://react.lightningdesignsystem.com/components/buttons/ . I think there are other examples in the library. The easiest way to check is to run `npm build:docs` and make sure this component shows up.

@davidlygagnon I'd recommend running the doc site consuming this branch locally to make sure all the comments and props show up correctly--or maintainers can fix this up, since the doc site isn't public. Either way is fine. As you know, the whole library/doc site can be finicky. Kevin and I spent some time with Setup Assistant yesterday.

Reply:
I have added multi-line comments right before the components declaration but haven't removed the `propTypes`. It is as done in `BuilderHeader` component that I was using as a reference. Let me know if this needs further updation.

Reply:
I'm referring to doc comments only. Please see https://github.com/salesforce/design-system-react/blob/master/components/builder-header/nav-dropdown.jsx#L16

which shows up on the doc site here: 
<img width="595" alt="Screen Shot 2019-06-13 at 1 57 59 PM" src="https://user-images.githubusercontent.com/1290832/59463489-4f5b8000-8de3-11e9-8631-662d91645b0d.png">

Please add a link to the doc site Button there.

Reply:
Cool - I'll double check by building the doc site locally.